views/play.erb

```diff
@@ -14,10 +14,10 @@
     <br>
     <br>
     <div align="center">
-        <form>
-           <input id="rock" name="rock" type="button" value="Rock">
```

Comment:
I see you removed the rock paper scissors from here, is there a reason for this?